client/components/domains/register-domain-step/index.jsx

```diff
@@ -166,6 +166,12 @@ var RegisterDomainStep = React.createClass( {
 	},
 
 	searchForm: function() {
+		var placeholderText = this.translate( 'Enter a domain or keyword', { textOnly: true } );
```

Comment:
You can get rid of all these `{ textOnly: true }` parameters - they are not needed any more.